<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-22T13:13:22 CDT" name="Suite" finished-at="2021-06-22T13:13:22 CDT" duration-ms="386">
    <groups>
    </groups>
    <test started-at="2021-06-22T13:13:22 CDT" name="Test1" finished-at="2021-06-22T13:13:22 CDT" duration-ms="126">
      <class name="Package.CalculatorTests">
        <test-method signature="MultiplyTest_MultipleSetOfNumbersXML_ValidCase(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Package.CalculatorTests@5b94b04d]" started-at="2021-06-22T13:13:22 CDT" name="MultiplyTest_MultipleSetOfNumbersXML_ValidCase" finished-at="2021-06-22T13:13:22 CDT" duration-ms="31" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[25]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MultiplyTest_MultipleSetOfNumbersXML_ValidCase -->
      </class> <!-- Package.CalculatorTests -->
    </test> <!-- Test1 -->
    <test started-at="2021-06-22T13:13:22 CDT" name="Test2" finished-at="2021-06-22T13:13:22 CDT" duration-ms="10">
      <class name="Package.CalculatorTests">
        <test-method signature="MultiplyTest_MultipleSetOfNumbersXML_ValidCase(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Package.CalculatorTests@6e509ffa]" started-at="2021-06-22T13:13:22 CDT" name="MultiplyTest_MultipleSetOfNumbersXML_ValidCase" finished-at="2021-06-22T13:13:22 CDT" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[12]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MultiplyTest_MultipleSetOfNumbersXML_ValidCase -->
      </class> <!-- Package.CalculatorTests -->
    </test> <!-- Test2 -->
    <test started-at="2021-06-22T13:13:22 CDT" name="Test3" finished-at="2021-06-22T13:13:22 CDT" duration-ms="24">
      <class name="Package.CalculatorTests">
        <test-method signature="MultiplyTest_MultipleSetOfNumbersXML_ValidCase(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Package.CalculatorTests@313b2ea6]" started-at="2021-06-22T13:13:22 CDT" name="MultiplyTest_MultipleSetOfNumbersXML_ValidCase" finished-at="2021-06-22T13:13:22 CDT" duration-ms="5" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[8]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[5]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual and Expected do not match expected [5] but found [16]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual and Expected do not match expected [5] but found [16]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at Package.CalculatorTests.MultiplyTest_MultipleSetOfNumbersXML_ValidCase(CalculatorTests.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MultiplyTest_MultipleSetOfNumbersXML_ValidCase -->
      </class> <!-- Package.CalculatorTests -->
    </test> <!-- Test3 -->
  </suite> <!-- Suite -->
</testng-results>
